test/JsonApiDotNetCoreMongoDbExampleTests/IntegrationTests/Sorting/SortTests.cs

```diff
@@ -0,0 +1,202 @@
+using System.Collections.Generic;
+using System.Net;
+using System.Threading.Tasks;
+using FluentAssertions;
+using JsonApiDotNetCore.Serialization.Objects;
+using JsonApiDotNetCoreMongoDbExample;
+using JsonApiDotNetCoreMongoDbExample.Models;
+using MongoDB.Driver;
+using Xunit;
+using Person = JsonApiDotNetCoreMongoDbExample.Models.Person;
+
```

Comment:
Please also add all tests from ResourceDefinitionQueryCallbackTests.cs, except `Include_from_resource_definition_has_blocked_capability`.

These influence the query being sent to the database, so it hits MongoDb repository logic.

Reply:
I'm still failing the two tests that filter by resource metadata so this one is to be considered partially addressed too.

I'm referring to the following tests:
* Queryable_parameter_handler_from_resource_definition_and_query_string_filter_are_applied
* Queryable_parameter_handler_from_resource_definition_is_applied

Reply:
They fail because the following needs to be added: https://github.com/json-api-dotnet/JsonApiDotNetCore/blob/master/src/JsonApiDotNetCore/Repositories/EntityFrameworkCoreRepository.cs#L93-L103

Reply:
Looked at `EntityFrameworkCoreRepository` in the original repo and figured out what was missing, just added a commit adding support for custom query parameters so this one is finally addressed.